### PR TITLE
Added docs on Clair v4 per PROJQUAY-332

### PR DESCRIPTION
**Pull request by chrisnegus**
Created a new section on adding Clair v4 to an existing Quay 3.3 + Clair v2 setup.

### CODE REVIEW

File: modules/proc_manage-clairv4.adoc [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+            - name: POSTGRES_DB
+              value: "clair"
+            - name: POSTGRES_PASSWORD
+              value: "postgres"
+          volumeMounts:
+            - name: postgres-data
+              mountPath: "/var/lib/postgres/data"
+---
+apiVersion: v1
+kind: PersistentVolumeClaim <1>
```
Comment (dongboyan77) [inline modules/proc_manage-clairv4.adoc:102]: remove `<1>`

File: modules/proc_manage-clairv4.adoc [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+[source,yaml]
+----
+SECURITY_SCANNER_V4_ENDPOINT: http://clairv4 <1>
+SECURITY_SCANNER_V4_NAMESPACE_WHITELIST: <2>
+  - "clairv4-org"
+  - "foo-org"
+----
+<1> Identify the Clair v4 service endpoint
+<2> Replace `clair4-org` and `foo-org` with name of Organizations
+in your {productname} cluster you want to use Clair v4 scanning.
```
Comment (dongboyan77) [inline modules/proc_manage-clairv4.adoc:230]: Should be namespaces contain organizations and user

File: modules/proc_manage-clairv4.adoc [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+. Modify the `config.yaml` file for your {productname} deployment to add the following
+two entries at the end:
++
+[source,yaml]
+----
+SECURITY_SCANNER_V4_ENDPOINT: http://clairv4 <1>
+SECURITY_SCANNER_V4_NAMESPACE_WHITELIST: <2>
+  - "clairv4-org"
+  - "foo-org"
+----
```
Comment (dongboyan77) [inline modules/proc_manage-clairv4.adoc:227]: Suggest to add `FEATURE_SECURITY_SCANNER: true`

**Comment by dongboyan77** [comment]
We can add a part of mounting Quay self-signed certificate into clair v4.
1. Create a secret contains self-certificate `oc create secret generic quay-certs --from-file=quay.crt`
2. Create clair v4 deployment `oc create -f https://gist.github.com/dongboyan77/65d7527581a8f208b2bc19e57c877376`
The key point is mount secret to path `/etc/ssl/certs` in pod

File: modules/proc_manage-clairv4.adoc [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+development.
+
+== What is Clair v4?
+
+Technically, Clair v4 is a set of micro services that can be used with {productname}
+to perform vulnerability scanning of container images associated with a set of
+Linux operating systems. The micro services design of Clair v4 makes it
+appropriate to run in a highly scalable configuration, where
+components can be scaled separately as appropriate for enterprise environments.
+The Clair v4 link:https://github.com/quay/openshift-demo/blob/master/clairv4/config.yaml[config.yaml]
```
Comment (ldelossa) [inline modules/proc_manage-clairv4.adoc:18]: I'm not sure linking to the config here is that valuable. I do have this document:
https://app.lucidchart.com/invitations/accept/7c5aa5a0-6ff6-4f72-ac45-fa3669c5cb77

I believe it is public. But ideally you can export this document and make it part of the prose

File: modules/proc_manage-clairv4.adoc (hunk 1 above)
Comment (chrisnegus) [reply modules/proc_manage-clairv4.adoc:102]: The <1> refers to the annotation under the yaml file.

File: modules/proc_manage-clairv4.adoc (hunk 2 above)
Comment (chrisnegus) [reply modules/proc_manage-clairv4.adoc:230]: Okay. I'll say: "<2> Replace `clair4-org` and `foo-org` with namespaces (organizations and
users) in your {productname} cluster you want to use Clair v4 scanning."

File: modules/proc_manage-clairv4.adoc (hunk 3 above)
Comment (chrisnegus) [reply modules/proc_manage-clairv4.adoc:227]: Done.

File: modules/proc_manage-clairv4.adoc (hunk 4 above)
Comment (chrisnegus) [reply modules/proc_manage-clairv4.adoc:18]: I'll cut the link to config.yaml for now. I can't get to the lucidchart document. We can revisit later.

**Comment by dongboyan77** [comment]
@chrisnegus I am not sure if you see my last comment about adding certificate into clair v4

**Pull request merged**